include/mbedtls/ctr_drbg.h

```diff
@@ -49,10 +49,10 @@
 
 #define MBEDTLS_CTR_DRBG_BLOCKSIZE          16 /**< The block size used by the cipher. */
 
-#if !defined(MBEDTLS_CTR_DRBG_USE_128_BIT_KEY) /* Use default 256-bit key for CTR_DRBG. */
-#define MBEDTLS_CTR_DRBG_KEYSIZE            32 /**< The key size used by the cipher. */
-#else
+#if defined(MBEDTLS_CTR_DRBG_USE_128_BIT_KEY) /* Use default 256-bit key for CTR_DRBG. */
```

Comment:
“Use default 256-bit key for CTR_DRBG” doesn't correspond to the case `defined(MBEDTLS_CTR_DRBG_USE_128_BIT_KEY)`. It would be better if the typeset documentation reflected the choice of key size anyway. Suggestion:
```
#if defined(MBEDTLS_CTR_DRBG_USE_128_BIT_KEY)
#define MBEDTLS_CTR_DRBG_KEYSIZE            16 /**< The key size used by the cipher (compile-time choice: 128 bits). */
￼ #else
￼ #define MBEDTLS_CTR_DRBG_KEYSIZE            32 /**< The key size used by the cipher (compile-time choice: 256 bits). */
￼ #endif
```

Reply:
good point, your version looks better. I'm amending the commit to reduce noise.